projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/sample/Sample.java

```diff
@@ -54,6 +58,8 @@ private Sample(SampleId id, SampleCode sampleCode, BatchId assignedBatch, String
     this.sampleOrigin = sampleOrigin;
     this.biologicalReplicateId = replicateReference;
     this.assignedBatch = assignedBatch;
+    this.analysisType = analysisType.orElse("");
+    this.comment = comment.orElse("");
```

Comment:
I think the sample itself should have `null` values in these fields as they are optional for the sample.

Reply:
the sample does have nullable fields and optional getters. the sample registration request can have null values. that is fine by me. 
In the initial review, I missed the record for the constructor.